modules/apigee_edge_teams/apigee_edge_teams.install

```diff
@@ -59,3 +61,78 @@ function apigee_edge_teams_requirements($phase) {
 
   return $requirements;
 }
+
+/**
+ * Implements hook_install().
+ */
+function apigee_edge_teams_install() {
+  if (\Drupal::moduleHandler()->moduleExists('user')) {
+    $authenticated_user_permissions = [
+      'accept own team invitation',
+      'decline own team invitation',
+    ];
+    user_role_grant_permissions(RoleInterface::AUTHENTICATED_ID, $authenticated_user_permissions);
+  }
+}
+
+
+/**
+ * Install team_invitation entity, dependencies and config.
+ */
+function apigee_edge_teams_update_8701() {
+  // Install 'team_invitation' entity.
+  \Drupal::entityTypeManager()->clearCachedDefinitions();
+  $entity_definition_update_manager = \Drupal::entityDefinitionUpdateManager();
+  $entity_definition_update_manager->installEntityType(\Drupal::entityTypeManager()->getDefinition('team_invitation'));
+
+  // Install 'views' module.
+  \Drupal::service('module_installer')->install(['views']);
+
+  // Install new config.
+  $settings_to_sync = [
+    'team_invitation_expiry',
+    'team_invitation_email_existing',
+    'team_invitation_email_new',
+  ];
+  /** @var \Drupal\Core\Config\StorageInterface $config_storage */
+  $config_storage = \Drupal::service('config.storage');
+  $module_path = drupal_get_path('module', 'apigee_edge_teams');
+  $source = new FileStorage("$module_path/config/install");
+  $new_team_settings = $source->read('apigee_edge_teams.team_settings');
+  $team_settings = $config_storage->read('apigee_edge_teams.team_settings');
+  foreach ($settings_to_sync as $setting) {
+    $team_settings[$setting] = $new_team_settings[$setting];
+  }
+  $config_storage->write('apigee_edge_teams.team_settings', $team_settings);
+
+  $team_invitations_view = $source->read('views.view.team_invitations');
```

Comment:
'views.view.team_invitations' source is 'config/optional/views.view.team_invitations.yml'. but $source referring '/config/install'. I think,  'views.view.team_invitations' need a new source variable.

Reply:
This is still wip. Nice catch though. We'll fix it.